src/scss/fotorama.scss

```diff
@@ -18,7 +18,7 @@ $blue: mix(#00d1ff, #008ed6);
   }
 }
 
-.fullscreen {
+.fotorama__fullscreen {
```

Comment:
agree, but right now i want to demonstrate what part of src must be changed for fix it.